src/marks/contour.js

```diff
@@ -1,7 +1,7 @@
-import {blur2, contours, geoPath, map, max, min, nice, range, ticks, thresholdSturges} from "d3";
+import {blur2, contours, geoPath, max, min, nice, range, ticks, thresholdSturges} from "d3";
 import {Channels} from "../channel.js";
 import {create} from "../context.js";
-import {labelof, identity, arrayify} from "../options.js";
+import {labelof, identity, arrayify, map} from "../options.js";
```

Comment:
Oops, this fixes the calls here: 😳 

https://github.com/observablehq/plot/blob/1186b5b19c04e8a52cfec399743ad43fb44ab25b/src/marks/contour.js#L125-L126